app/src/components/tables/BlockTable/BlocksTable.test.tsx

```diff
@@ -0,0 +1,204 @@
+import React from 'react';
+import { screen } from '@testing-library/react';
+import { Table } from 'src/components';
+import {
+  getMockBlocks,
+  mockBlocksTableColumns,
+  getMockBlocksTableFooter,
+  getMockBlocksTableHeader,
+  mockBlocksTableOptions,
+} from 'src/mocks/mock-blocks-table';
+import { formatDate, truncateHash } from 'src/utils';
+import { render } from '../../../test-utils';
+
+const blocks = getMockBlocks();
+const mockBlocksTableFooter = getMockBlocksTableFooter();
+const mockBlocksTableHeader = getMockBlocksTableHeader();
+const totalPages = Math.ceil(
+  blocks.length / mockBlocksTableOptions.pagination.pageSize,
+);
+const placeholderData = {
+  header: {
+    height: 0,
+    era_id: 0,
+    timestamp: '2023-06-05T17:06:44.864Z',
+  },
+  body: {
+    proposer:
+      '017d96b9a63abcb61c870a4f55187a0a7ac24096bdb5fc585c12a686a4d892009e',
+    deploy_hashes: [],
+    transfer_hashes: [],
+  },
+  hash: '52f7c16323868f73343335f26a484aed0067a3e769dc9187dbae6a305e2b59f3',
+};
+
+describe('BlocksTable', () => {
+  it('should render the BlocksTable', () => {
+    render(
+      <Table
+        header={mockBlocksTableHeader}
+        columns={mockBlocksTableColumns}
+        currentPageSize={mockBlocksTableOptions.pagination.pageSize}
+        isLastPage={totalPages === mockBlocksTableOptions.pagination.pageNum}
+        data={blocks}
+        footer={mockBlocksTableFooter}
+        tableBodyLoading={false}
```

Comment:
FYI: this prop is optional and should default to false.  If it doesn't then that's a bug and we should fix it :)